lib/blocktree/blocktree.go

```diff
@@ -49,34 +47,25 @@ func NewBlockTreeFromRoot(root *types.Header) *BlockTree {
 	}
 
 	return &BlockTree{
-		root:      n,
-		leaves:    newLeafMap(n),
-		nodeCache: make(map[Hash]*node),
-		runtime:   &sync.Map{},
+		root:    n,
+		leaves:  newLeafMap(n),
+		runtime: &sync.Map{},
 	}
 }
 
-// GenesisHash returns the hash of the genesis block
-func (bt *BlockTree) GenesisHash() Hash {
```

Comment:
Is this function just not something we need anymore?

Reply:
yeah it was unused